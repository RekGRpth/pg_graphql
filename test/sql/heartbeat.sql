select
    graphql.resolve($${ utcNow: heartbeat }$$) -> 'data' ->> 'utcNow' like '2%'
