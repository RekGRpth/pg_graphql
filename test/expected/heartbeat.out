select
    graphql.resolve($${ utcNow: heartbeat }$$) -> 'data' ->> 'utcNow' like '2%'
 ?column? 
----------
 t
(1 row)

